File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1582.py
from math import gcd

while True:
    try:
        l = [int(i) for i in input().split()]
        a = l[0]
        b = l[1]
        c = l[2]
        
        if(b > a and b > c):
            temp = a
            a = b
            b = temp
        elif(c > a and c > b):
            temp = a
            a = c
            c = temp
        
        triplaP = False
        triplaPP = False
        
        if(a*a == b*b + c*c):
            triplaP = True
            if(gcd(gcd(a, b), c) == 1): triplaPP = True
        
        if(triplaP):
            if(triplaPP): print("tripla pitagorica primitiva")
            else: print("tripla pitagorica")
        else: print("tripla")
        
    except:
        break

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1582.cpp
#include <iostream>
#include <algorithm>

using namespace std;

bool pitagoras(int a, int b, int c){
    if(a*a + b*b == c*c) return true;
    return false;
}

int main(){
    int a, b, c, temp;
    while(cin >> a >> b >> c){
        if(b > a && b > c){
            temp = a;
            a = b;
            b = temp;
        } else if(c > a && c > b){
            temp = a;
            a = c;
            c = temp;
        }
        
        bool triplaP = false, triplaPP = false;
        
        
        if(a*a == b*b + c*c){
            triplaP = true;
            if(__gcd(__gcd(a, b), c) == 1) triplaPP = true;
        }
        
        if(triplaP){
            if(triplaPP) cout << "tripla pitagorica primitiva" << endl;
            else cout << "tripla pitagorica" << endl;
        } else cout << "tripla" << endl;

    }
    
    return 0;
}